crcsim/experiment/prepare.py

```diff
@@ -144,8 +165,13 @@ def create_scenarios() -> List:
     low_surveillance_freq_mild = 10
     low_surveillance_freq_severe = 2
     low_surveillance_end_age = 80
```

Comment:
We don't need the FQHC-specific scenarios or the sensitivity tests from the previous experiment in this one.